spec/unit/client_spec.rb

```diff
@@ -284,18 +312,19 @@ def fully_qualified_api_endpoint
       :score_response => action_response_json
     }
 
-    stub_request(:post, "https://api.siftscience.com/v203/events?return_action=true").
-      to_return(:status => 200, :body => MultiJson.dump(response_json), :headers => {"content-type"=>"application/json; charset=UTF-8","content-length"=> "74"})
+    stub_request(:post, "https://api.siftscience.com/v204/events?return_action=true")
+      .to_return(:status => 200, :body => MultiJson.dump(response_json),
+                 :headers => {"content-type"=>"application/json; charset=UTF-8",
+                              "content-length"=> "74"})
 
     event = "$transaction"
     properties = valid_transaction_properties
-    response = Sift::Client.new(api_key).track(event, properties, nil, nil, nil, nil, true)
+    response = Sift::Client.new(:api_key => api_key)
+               .track(event, properties, :return_action => true)
     expect(response.ok?).to eq(true)
     expect(response.api_status).to eq(0)
     expect(response.api_error_message).to eq("OK")
     expect(response.body["score_response"]["actions"].first["entity"]["type"]).to eq("USER")
   end
 
-
```

Comment:
This functionality has been tested manually.  I may add automated tests tomorrow.